src/applications/personalization/profile360/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -41,37 +43,62 @@ const teardown = () => {
 };
 
 describe('actions/paymentInformation', () => {
-  beforeEach(setup);
-  afterEach(teardown);
+  describe('when global `ga` is set up correctly', () => {
```

Comment:
sorry y'all nesting these existing tests inside a new `describe()` made the diff noisy.